internal/cmd/load-balancer/list/list.go

```diff
@@ -133,6 +134,7 @@ func outputResult(p *print.Printer, outputFormat string, loadBalancers []loadbal
 		for i := range loadBalancers {
 			l := loadBalancers[i]
 			table.AddRow(*l.Name, *l.Status, *l.ExternalAddress, len(*l.Listeners), len(*l.TargetPools))
+			table.AddSeparator()
```

Comment:
Same here, we don't usually have separators for horizontal tables

Reply:
![image](https://github.com/stackitcloud/stackit-cli/assets/28832884/2d327690-151b-4100-a81b-b745b7599a84)